app/views/analytics/districts/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="dashboard">
   <nav class="breadcrumb">
-    <%= link_to 'All districts', admin_root_path %>
+    <%= link_to 'All districts', email_authentication_root_path %>
```

Comment:
`email_authentication_root_path` isn't a great name for `organizations#index`, can we call this something else?

Reply:
This is generated by devise.